ui/tests/acceptance/oidc-auth-method-test.js

```diff
@@ -91,6 +91,7 @@ module('Acceptance | oidc auth method', function (hooks) {
       cancelTimers();
     }, 50);
     await click('[data-test-auth-submit]');
+    await waitUntil(() => find('.nav-user-button button'));
```

Comment:
Okay! I was wondering about this - since I noticed we don't use timeouts in our tests much? So something like this? 

```suggestion
    await waitUntil(() => find('.nav-user-button button'), { timeout: 2000 });
```

It looks like there's already a default of 1000

Reply:
Interesting 🤔 the waitUntil definitely has borked our tests in the past. Maybe we override the timeout somewhere in config?
But yes the suggested commit is what I had in mind!